source/class/qx/log/appender/NodeConsole.js

```diff
@@ -37,6 +37,32 @@ qx.Class.define("qx.log.appender.NodeConsole", {
      */
     __ERR : null,
 
+    /**
+     * Whether to use color codes
+     */
+    __useColors: false,
```

Comment:
What's about a property for this purpose?

Reply:
It is a static class so the whole property boilerplate would have to be manually set up. Can we do without, given that there doesn't really seem to be a need for a getter here?